omero/developers/Web/LinkingFromWebclient.txt

```diff
@@ -132,7 +132,17 @@ attribute to the options:
 
 ::
 
-    $ bin/omero config append omero.web.open_with '["My viewer", "url_name"], {"supported_objects": ["image"], "target": "_blank"}]'
+    $ bin/omero config append omero.web.open_with '["xyz_viewer", "url_name"], {"supported_objects": ["image"], "target": "_blank"}]'
+
+UI Label
+^^^^^^^^
+
+If a "label" is specified in the options object, this will be used as the display label in the webclient context menu
```

Comment:
An explanation is needed about what "xyz_viewer" represents as opposed to the additional label "X-Y-Z viewer". Creating an example mentioning the "xyz_viewer" value for OMERO.figure would be hightly desirable.